framework/test/unit/application.spec.ts

```diff
@@ -166,6 +166,31 @@ describe('Application', () => {
 				"Lisk validator found 1 error[s]:\nMissing property, should have required property 'communityIdentifier'",
 			);
 		});
+
+		it('should throw if invalid forger is provided', () => {
+			// Arrange
+			const invalidConfig = objects.mergeDeep({}, config, {
+				forging: {
+					delegates: [
+						{
+							encryptedPassphrase:
+								'0dbd21ac5c154dbb72ce90a4e252a64b692203a4f8e25f8bfa1b1993e2ba7a9bd9e1ef1896d8d584a62daf17a8ccf12b99f29521b92cc98b74434ff501374f7e1c6d8371a6ce4e2d083489',
+							address: '9cabee3d27426676b852ce6b804cb2fdff7cd0b5',
+							hashOnion: {
+								count: 0,
+								distance: 0,
+								hashes: [],
+							},
+						},
+					],
+				},
+			});
+
+			// Act & Assert
+			expect(() => Application.defaultApplication(genesisBlockJSON, invalidConfig)).toThrow(
+				'Lisk validator found 4 error[s]',
```

Comment:
Please add exception for a particular error. Above 4 errors can be anything else non-related as well.